core-addon/Core.js

```diff
@@ -558,13 +561,15 @@ module.exports = class Core {
    */
   async _sendStateUpdateToUI() {
     let enrolled = !!(await this._storage.getRallyID());
```

Comment:
We already only set the `enrolled` status if after user goes through the onboarding and consents to join rally. Isn't this enough?

Reply:
It isn't. We only want the arrow to appear on the very first load of the welcome page.